### ISSUE TITLE
NAVTEX Decoding

### ISSUE THREAD

**Issue by ja9mat** [RaspSDR/server#49]
WSPR and FAX are decoded well.
But the NAVTEX is not decoded. What Should I do? An additional setting needed?

**Comment by dannybergen** [comment]
I am able to decode the 3 NAVTEX frequencies listed in the FSK extension. But none og these work in the NAVTEX extension. The NAVTEX extension does not function as far as I have experienced.

**Comment by ja9mat** [comment]
Thanks. I will try your way!2024/11/15 18:47、dannybergen ***@***.***>のメール:﻿
I am able to decode the 3 NAVTEX frequencies listed in the FSK extension. But none og these work in the NAVTEX extension. The NAVTEX extension does not function as far as I have experienced.

**Comment by ja9mat** [comment]
Disabled map but not changed....

**Comment by howard0su** [comment]
did you try the latest firmware 1115 which contains the fix?

**Comment by ja9mat** [comment]
Yes my installed version is v2024.1115, built Nov 15 2024 03:59:14.
I set display options to message not to map, is this OK?

**Comment by howard0su** [comment]
Yes, it is correct.

**Comment by ja9mat** [comment]
Thanks, but I doubt another setting needed I don't know.

**Comment by ja9mat** [comment]
Now testing but nothing appear...as an attached movie file.

https://github.com/user-attachments/assets/30a564d6-9066-4db8-b78d-ed319d877563

**Comment by howard0su** [comment]
can you try:
1. redownload the zip file, and update it again. (omit config as you want to retain the configurations.)
2. Press Ctrl-F5 to force refresh the local javascript cache in browser.
3. If this doesn't work, can you open developer console (normally press F12) to check "Console" tab to see if there is any error?

Your recording shows it is still using an old version of javascript file.

**Comment by ja9mat** [comment]
Now works fine!
Ctrl+F5 not available so I redownload and rewrite then it's OK.
Thanks for your support.
